Add CFP info for KCSNA (#513)
kubernetes/contributor-site: content/en/events/2024/kcsna/_index.md

### Call for Session Proposals

> A [CFP] for sessions will be open through Sunday, September 15th.

Reviewer: ```suggestion
A [CFP] for sessions is opened through Sunday, September 15th.
```

> A [CFP](https://forms.gle/5RJBiKtuYsBDYAsVA) for sessions will be open through Sunday, September 15th.
> 
> [CFP]: (https://forms.gle/5RJBiKtuYsBDYAsVA)

Reviewer 2: I'd pick one or the other rather than both, but I don't know that the build system actually cares. Thanks for the update!

```suggestion
A [CFP] for sessions will be open through Sunday, September 15th.

[CFP]: (https://forms.gle/5RJBiKtuYsBDYAsVA)
```

or

```
A [CFP](https://forms.gle/5RJBiKtuYsBDYAsVA) for sessions will be open through Sunday, September 15th.
```

Author: Ok, let me update. I haven't done a ton with markdown like this and I saw this in a few of the previous files and based this on that.
